.hooks/pre_commit_stages/stage_eslint.py

```diff
@@ -1,25 +1,41 @@
 """
 Contains class 'ESLint' to run ESLint on JavaScript files
 """
+from subprocess import PIPE, Popen  # nosec
+
 from .stage_base import Stage
-from subprocess import Popen, PIPE  # nosec
 
 
 class ESLint(Stage):
     """
     Stage for running ESLint on JavaScript files
     """
+
     file_formats = ["js"]
 
     def run(self) -> None:
         for file in self.files:
             if file.split(".")[-1] in self.file_formats:
                 print(f"Running ESLint for {file}")
                 # pylint: disable=consider-using-with
-                process = Popen(["npm", "install", "eslint@8.10.0", "&&",    # nosec
-                                 "npm", "install", "eslint-config-airbnb-base@15.0.0", "&&",
-                                 "npx", "eslint", file], stdout=PIPE,
-                                stderr=PIPE, shell=True)
+                process = Popen(
+                    [
+                        "npm",
+                        "install",
+                        "eslint@8.10.0",
+                        "&&",  # nosec
```

Comment:
> Better after the closing bracket, I see that these annotations have not been optimally positioned from the start, I will fix this

Interestingly that did not seem to work, so either in the line of `Popen` or in the line actually causing the issue (`shell=True`), now it is the latter, so future reformatting should not mess with these annotations again

Reply:
For me placing the `# nosec` comment in the line of the function call (open bracket) does work.

Reply:
Fixed

Reply:
> For me placing the # nosec comment in the line of the function call (open bracket) does work.

Yes, I meant the closing brackets, for Pylint if I remember correctly you can also add such annotation after the closing bracket

Reply:
I did not realise that you already fixed those and made commits. (I did not run git pull). Thanks for the fixes!
When a comment/discussion is fixed it would be nice to also say so here in the comment or resolve the conversation.

Reply:
Ah, alright 👍 And yeah, I resolved all other `# no sec` -related comments here, but wanted to keep this one open so you can see why I did these commits and comment on it directly, if you would do it differently for example. With 
> now it is the latter 

I wanted to express that I changed it - but I agree, that this is not very clear and I should have linked the related commit. So it seems like I reached kind of the opposite and caused some confusion 😅 - sorry for that!